### PR TITLE
Issue329 issue handling workflow to docs

### PR DESCRIPTION
**Pull request by jonbergh**
Suggested solution to issue #329. Please check and comment.

### CODE REVIEW

File: docs/source/conventions.rst [hunk 1]
```diff
@@ -20,7 +20,22 @@ We adopt the following system for branching and merging:
    5. Branch only from master or from develop.
    6. Create pull requests for your branches and **always** assign a reviewer to merge, delete the branch, and close the issue (this is easy in github)
 
-Workflow for hotfix branches based on master. (Workflow for issue branches based on develop is similar):
+How to report and handle new issues (bugs, improvements, new features, etc.)
+----------------------------------------------------------------------------
+
+If you discover a bug in Nansat or you would like to suggest improvements to Nansat, the following procedure should be followed:
+
+1. Check that noone else has reported the same issue at https://github.com/nansencenter/nansat/issues
```
Comment (akorosov) [inline docs/source/conventions.rst]: replace 'noone' with 'no one'

File: docs/source/conventions.rst [hunk 2]
```diff
@@ -20,7 +20,22 @@ We adopt the following system for branching and merging:
...
+2. If not, add a new issue at https://github.com/nansencenter/nansat/issues
+3. Fork Nansat and clone your own fork locally
```
Comment (akorosov) [inline docs/source/conventions.rst]: Add: 3. If Nansat repository if not accessible for writing, fork ...

**Comment by akorosov** [comment]
@jonbergh , can you fix these minor corrections?

**Comment by jonbergh** [comment]
OK! I'm on it!

**Pull request merged**